src/main/java/com/robin/hoteless/ProgramEntryPoint.java

```diff
@@ -4,8 +4,9 @@
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 
 @SpringBootApplication
-public class Application {
+public class ProgramEntryPoint {
+
     public static void main(String[] args) {
-        SpringApplication.run(Application.class, args);
+        SpringApplication.run(ProgramEntryPoint.class, args);
```

Comment:
Haha this may be one of your issues - this should always be named ``Application``, don't rename it ;)